compute/admin_guide/waas/deploy_waas/deployment_vpc_mirroring.adoc

```diff
@@ -1,5 +1,7 @@
 == Deploy WAAS Out-Of-Band with VPC Traffic Mirroring
 
+NOTE: VPC Traffic Mirroring requires a minimum 22.06 version of both the Console and Defenders.
```

Comment:
@e-shuster On this page, we refer to the agent as an Observer, so do you think we might need to edit the above sentence?